client/lib/plans/constants.js

```diff
@@ -651,6 +653,12 @@ export const FEATURES_LIST = {
 		getDescription: () => i18n.translate( 'Install custom plugins on your site.' )
 	},
 
+	[ FEATURE_UPLOAD_THEMES ]: {
+		getSlug: () => FEATURE_UPLOAD_THEMES,
+		getTitle: () => i18n.translate( 'Upload themes' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Uploaded themes' )` *ES Score: 7.25*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/upload-upgrade-nudge)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).